deep_quoridor/src/arena.py

```diff
@@ -0,0 +1,91 @@
+from typing import Optional
+from quoridor_env import env
+
```

Comment:
can you add a brief description on each "interface" here on what the intention you had in mind is? It will be useful to continue constructing these with the same idea in mind